examples/with-react-intl/lang/en.json

```diff
@@ -1,7 +1,7 @@
 {
-  "title": "React Intl Next.js Example",
-  "nav.home": "Home",
-  "nav.about": "About",
-  "description": "An example app integrating React Intl with Next.js",
-  "greeting": "Hello, World!"
+  "11754": "An example app integrating React Intl with Next.js",
+  "65a8e": "Hello, World!",
+  "8cf04": "Home",
+  "8f7f4": "About",
+  "9c817": "React Intl Next.js Example"
```

Comment:
Why is having a generated id better here?

Reply:
Manual ID typically doesn't scale in larger applications since it's fairly easy to cause collision. We generally do not recommend it https://formatjs.io/docs/getting-started/message-extraction#automatic-id-generation